test/cpp/jit/test_gpu.cpp

```diff
@@ -15363,6 +15363,130 @@ TEST(NVFuserTest, TestBackOffInnerBroadcast3_CUDA) {
   TORCH_CHECK(tv3->getMaxProducerPosition() == 3);
 }
 
+TEST(NVFuserTest, TestSimpleWarp_CUDA) {
```

Comment:
Not sure it's intentional, not splitting tv3 like tv1 seems to make this test interesting. The thread-block size is 128 because of the lack of tv3 split, and the test is failing because the reduction is a warp reduction, but the broadcast is not.

We probably need to check the size of the concrete domain of a broadcast axis.

Reply:
Yes I just cleaned up this test. 

The silent error does concern me a bit, and it was not limited to broadcast-related graphs. We could move this in a separate issue to discuss.

Reply:
I thought this only happens when fusing a reduction and a broadcast. Are you referring to something more general?

Reply:
Yes I think the earlier issue was that the schedule was binding inconsistent iterdomains to TIDx. I will try to see if I could do a minimal repro and discuss this case in a follow up.